.coveragerc

```diff
@@ -0,0 +1,10 @@
+[run]
+omit =
+  cnxml/tests/*
```

Comment:
skipping tests completely, rather than just relaxing line limits? Is this a good idea? Tests tend to accumulate cruft if not analyzed.